app/src/sys/win/command.c

```diff
@@ -7,7 +7,7 @@
 static int
 build_cmd(char *cmd, size_t len, const char *const argv[]) {
     // Windows command-line parsing is WTF:
-    // <http://daviddeley.com/autohotkey/parameters/parameters.htm#WINPASS>
+    // <https://daviddeley.com/autohotkey/parameters/parameters.htm#WINPASS>
```

Comment:
The page is not available over HTTPS (broken certificate + 404).

Reply:
Okay, then it would be better keeping original one. Thanks!!